modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/RowHandler.java

```diff
@@ -17,36 +17,75 @@
 
 package org.apache.ignite.internal.sql.engine.exec;
 
+import java.nio.ByteBuffer;
 import org.apache.ignite.internal.sql.engine.exec.row.RowSchema;
 import org.jetbrains.annotations.Nullable;
 
 /**
  * Universal accessor and mutator for rows. It also has factory methods.
  */
 public interface RowHandler<RowT> {
+    /**
+     * Extract appropriate field.
+     *
+     * @param field position.
+     * @param row object to be extracted from.
+     */
     @Nullable Object get(int field, RowT row);
 
+    /** Set incoming row field.
+     *
+     * @param field Field position to be processed.
+     * @param row which field need to be changed.
+     * @param val value should be set.
```

Comment:
Let's use same style for all parameters in the class.
Maybe parameters block is readable as raw test, but it looks inconsistent after processing by IDEA renderer or javadoc-tool.

Reply:
Thanks ! got it !